c/doc/run_sample_opensuse.md

```diff
@@ -0,0 +1,71 @@
+# openSUSE Setup
```

Comment:
Could you also add this to the list that references the run_on_samples_foo .md files?
It is in /c/readme.md file.

Thanks,
/Dan